mcmd/commands/import_.py

```diff
@@ -237,8 +237,14 @@ def _get_import_action(file_name):
 
 
 def _poll_for_completion(url):
+    def step_function(step):
+        """Increases time between polls with one second each time, with a maximum of 10 seconds."""
+        step += 1
+        return min(step, 10)
+
     polling.poll(lambda: get(url).json()['status'] != 'RUNNING',
-                 step=10,
+                 step=1,
```

Comment:
Shouldn't this be 0? Then you start polling at 1 second since you do += 1 at line 241.